perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -134,18 +134,21 @@ sub tweak_all_reports_map {
 
     # Bromley only subcategory stuff
     my %subcats = $self->subcategories;
-    my $filter = $c->stash->{filter_categories};
-    my @new_contacts;
-    foreach (@$filter) {
-        push @new_contacts, $_;
-        foreach (@{$subcats{$_->id}}) {
-            push @new_contacts, {
-                category => $_->{key},
-                category_display => (" " x 4) . $_->{name},
-            };
+    my $groups = $c->stash->{category_groups};
```

Comment:
I do wonder if a bit more of a comment here to explain this might be helpful as I was initially a bit confused till I went and looked at the Bromley config